package_test.go

```diff
@@ -496,6 +496,33 @@ func (s *PackageSuite) TestNulls(c *C) {
 }
 
 func (s *PackageSuite) TestValidGet(c *C) {
+	// Embedded/nested structs.
+	type E1 struct {
+		Street string `db:"street"`
+	}
+	type E2 struct {
+		ID int `db:"id"`
+	}
+	type E3 struct {
+		District string `db:"district"`
+		E2       E2
+	}
+	type EmbeddedStruct struct {
+		E1 E1
+		E3 *E3
+	}
+
+	// Struct with pointers.
+	type PointerStruct struct {
```

Comment:
Would it be good to call this `StructWithPointers`?

Reply:
Changed.